bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -571,6 +571,10 @@
 		<label>Alarm State</label>
 		<description>The alarm state of the intrusion detection system. Possible values are ALARM_OFF, PRE_ALARM, ALARM_ON,
 			ALARM_MUTED and UNKNOWN.</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Presence</tag>
```

Comment:
I find this one difficult, but what about
Alarm -> Read only data point indicating an alarm condition

```suggestion
			<tag>Alarm</tag> 
			<tag>Presence</tag>
```